### DISCUSSION TITLE
Add Disable Functionality to Controls

### DISCUSSION THREAD

**Discussion by jneekamp-schulz** [Clooos/Bubble-Card#1062] [Feature requests]
**Context**

In Home Assistant, cards are a powerful way to customize the user interface. However, there is currently no native way to enable or disable controls within cards dynamically based on the state of an entity. While this can be achieved through custom styling options, having a dedicated property for this functionality would significantly improve ease of use and clarity.

**Request**

It would be immensely beneficial to introduce a disabled property for controls in Home Assistant cards. This property could accept a boolean value derived from an entity's state, enabling or disabling the respective component dynamically. For example:

```
type: custom:bubble-card
entity: switch.example
disabled: '{{ states("input_boolean.disable_controls") == "on" }}'
```

**Benefits**

**1. Improved Usability:**

A dedicated disabled property simplifies configurations and avoids complex CSS manipulations.

**2. Clarity:**

Users can easily understand and manage control states via a direct and explicit configuration option.

**Comment by MrBearPresident** [comment]
Hey thanks for your proposal.
This has been asked already a couple of times I think.
I will look tomorrow of the are already some duplicates.

But I'm interested. I personaly don't think this is necesarry because. If I don't want to control it. I'll use a state button or I'll dynamicly hide the button.

Just disabling Controls will only cutter up my dashboards.
